.github/workflows/pdf-docs-generation.yml

```diff
@@ -0,0 +1,24 @@
+name: PDF Docs Generation
+
+on:
+  push:
```

Comment:
Should we not just generate this on main branch to?

Reply:
yes good reminder i removed it as am working on cicd branch before